api/__tests__/util/hoursUtilities.test.ts

```diff
@@ -18,8 +18,7 @@ describe("Hours Utilities Tests", () => {
 
             expect(updatedUser.isCheckedIn).toBeTruthy();
             expect(updatedUser.transactions.length).toBe(1);
-            expect(updatedUser.transactions[0].checkOutTime).toBeNull();
-            expect(updatedUser.transactions[0].hours).toBeNull();
+            expect(updatedUser.transactions[0].checkOut).toBeNull();
         });
 
         it("should check in a user that's not checked in, multiple transactions", () => {
```

Comment:
We should add another test case that has a user with multiple transactions already. Code coverage is showing a missing block because of that